var/spack/repos/builtin/packages/py-celery/package.py

```diff
@@ -63,12 +64,24 @@ class PyCelery(PythonPackage):
     depends_on("py-click-plugins@1.1.1:", when="@5.0.3:", type=("build", "run"))
```

Comment:
Python 3.8+ required now

Reply:
Many deps need to be updated:
```
billiard>=4.1.0,<5.0
kombu>=5.3.2,<6.0
vine>=5.0.0,<6.0
click>=8.1.2,<9.0
click-didyoumean>=0.3.0
click-repl>=0.2.0
click-plugins>=1.1.1
importlib-metadata>=3.6; python_version < '3.8'
backports.zoneinfo>=0.2.1; python_version < '3.9'
tzdata>=2022.7
python-dateutil>=2.8.2
```

Reply:
Can you sort these in the same order as `requirements/default.txt`? Also sort them from newest to oldest like we do for versions. Makes it easier to review and ensure nothing is missing.